### PR TITLE
Sb 106 send notification using twilio api

### PR DESCRIPTION
**Pull request by StudentCZ**
https://virgl.atlassian.net/jira/software/projects/SB/boards/1?selectedIssue=SB-106

We created the SMS function and put all the sensitive information in an ENV file including the API Key. We are able to send users SMS and is able to send a dummy message from our Twilio Phone number to a valid number.
Screen Shot 2023-01-11 at 6 42 44 PM

### CODE REVIEW

**Review by tonyxuan2021 — APPROVED** [review]
Looks good to me. Aaron seems can import and and then call SMS() when the condition is met at arriving logic.

**Review by chunloy — COMMENTED** [review]
Maybe we can add parameters to SEND_SMS() e.g. user_id, passenger's number and status? I'd like to be able to send a couple templated messages with the passenger's name.

For example, let's assume `status="not picked up"` and the bus is 1km away. I would like to be able to send a message like, "Hey Aaron, your bus will be arriving shortly."

If `status="waiting"` then I'd like to notify the passenger that the bus has arrived.

Not sure about the best way to store these messages, e.g. keep them in a separate file, or just factor them out in this file. Any ideas?

**Comment by yhus** [comment]
This looks good.

I guess one part I'm confused about is... shouldn't this be exposed as a public endpoint? Or do we think the frontend will never explicitly ask the backend to send a SMS?

File: send_sms.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from dotenv import load_dotenv
+from twilio.rest import Client
+load_dotenv()
+import os
+
+def SEND_SMS():
```
[the remark this answers is not in the document]
Comment (StudentCZ) [reply send_sms.py]: fixed.

File: send_sms.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from dotenv import load_dotenv
+from twilio.rest import Client
```
[the remark this answers is not in the document]
Comment (StudentCZ) [reply send_sms.py:2]: Yes

File: send_sms.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+load_dotenv()
+import os
+
+def SEND_SMS():
+  account_sid = os.environ['TWILIO_ACCOUNT_SID']
+  auth_token = os.environ['TWILIO_AUTH_TOKEN']
+  client = Client(account_sid, auth_token)
+  message = client.messages \
+                .create(
```
[the remark this answers is not in the document]
Comment (StudentCZ) [reply send_sms.py:17]: I have spoke and pair programmed with Aaron. We have made the necessary changes.

File: send_sms.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+from dotenv import load_dotenv
+from twilio.rest import Client
+load_dotenv()
+import os
+
+def SEND_SMS():
+  account_sid = os.environ['TWILIO_ACCOUNT_SID']
```
[the remark this answers is not in the document]
Comment (StudentCZ) [reply send_sms.py]: I will make a Jira ticket for this and address it going forward, We will approve first for now.

**Review by nicolepa — APPROVED** [review]

**Pull request merged**